civitai-downloader-v2/src/cli/main.py

```
@@ -114,623 +114,349 @@ def cli(ctx, config, verbose):
 @click.option('--verify', is_flag=True, help='Verify file integrity after download')
 @click.option('--scan-security', is_flag=True, help='Scan file for security threats')
 @click.option('--no-progress', is_flag=True, help='Disable progress bar')
-def download_command(url_or_id, output_dir, filename, verify, scan_security, no_progress):
+@click.option('--force', is_flag=True, help='Force download even if already downloaded')
+def download_command(url_or_id, output_dir, filename, verify, scan_security, no_progress, force):
     """Download a model by URL or ID."""
     
     async def run_download():
         try:
+            # Use config for default output directory
+            if not output_dir:
```

Ruff (0.12.2)

Local variable `output_dir` referenced before assignment

(F823)